app/schemas/com.duckduckgo.app.remotemessage.store.RemoteMessagingDatabase/1.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I think this file shouldn't have been generated in the `:app` module. It should be in the remote-messaging specific modules.

Reply:
The issue was on the build.gradle, it's now fixed.